docs/spec_draft.md

```diff
@@ -223,6 +224,48 @@ Produces a `result` tensor from a constant `value`.
 // %3: [(0.0, 1.0), (2.0, 3.0)]
 ```
 
+## stablehlo.cosine
+
+`stablehlo.cosine(operand) -> result`
+
+### Semantics
+
+Performs element-wise cosine operation on `operand` tensor and produces a `result` tensor.
+
+### Attributes
+
+| Name | Type |
+|-|-|
+| `value` | `tensor` of floating-point or complex element types |
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `value` | Constant n-dimensional data of floating-point, or complex element types |
```

Comment:
We should use tensor which is already defined as n dimensional data. Something like

`tensor of floating-point or complex element types`

Reply:
here and below

Reply:
Fixed